iset.mm

```diff
@@ -125588,6 +125608,328 @@ reduced fraction representation (no common factors, denominator
   $}
 
 
+$(
+###############################################################################
+  BASIC TOPOLOGY
+###############################################################################
+$)
+
+
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Metric spaces
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Topological definitions using the reals
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
```

Comment:
@jkingdon : The nesting order should be swapped, here: "Metric spaces" is a particular kind of "Topological definition using the reals", which I would rename "Real numbers in topology", since, like most sections, it is not only about definitions.